src/services/dex.js

```diff
@@ -1462,7 +1486,7 @@ export default {
   },
 
   withdrawSystemAsset(assetId, quantity, utxoTxHash, utxoIndex, tryCount = 1) {
-    return new Promise((resolve, reject) => {
+    return new Promise(async (resolve, reject) => {
```

Comment:
This is probably another candidate for breaking up. IMO.

Reply:
ditto